docs/development.rst

```diff
@@ -21,14 +21,6 @@ provide helpful development tools and to allow the test suite to run: ::
 During development, once Girder is started via ``python -m girder``, the server
 will reload itself whenever a Python file is modified.
 
-Girder's web-based client application is built using the `Grunt <http://gruntjs.com/>`_
-task running tool. When you run the ``npm install`` command during Girder's
-installation, it will run all of the grunt tasks required to build the web client.
-Grunt tasks are run with the ``grunt`` executable, which is installed under your Girder source
-directory in the ``./node_modules/.bin/`` directory. You could conveniently update the
-``PATH`` by running ``export PATH=$(pwd)/node_modules/.bin:$PATH`` -- once you do that,
-you can just type ``grunt`` in your shell to run tasks.
-
```

Comment:
If you remove this block, you should probably remove the note below on Grunt installation.